presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -267,6 +267,40 @@ and a file system location of ``s3://test_bucket/test_schema/test_table``:
         location = 's3://test_bucket/test_schema/test_table')
     )
 
+Caching Support
+----------------
+
+Manifest File Caching
+^^^^^^^^^^^^^^^^^^^^^^
+
+Starting from Iceberg version 1.1.0, Apache Iceberg provides a mechanism to cache the contents of Iceberg manifest files in memory. This manifest caching feature helps
+to reduce repeated reads of small Iceberg manifest files from remote storage.
+
+.. note::
+
+    Currently, Manifest file caching is supported for Hadoop & Nessie catalogs from Presto Iceberg Connector.
```

Comment:
```suggestion
    Currently, manifest file caching is supported for Hadoop and Nessie catalogs in the Presto Iceberg connector.
```
Some minor revisions, mostly capitalization. Suggest "and" for "&" to avoid Latin abbreviations (such as e.g, i.e., and others). See the [GitLab docs style guide recommended word list entry](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html) for &.